include/tvm/runtime/threading_backend.h

```diff
@@ -64,21 +85,24 @@ class ThreadGroup {
   enum AffinityMode : int {
     kBig = 1,
     kLittle = -1,
+    kSpecify = -2,
   };
 
   /*!
    * \brief configure the CPU id affinity
    *
    * \param mode The preferred CPU type (1 = big, -1 = little).
    * \param nthreads The number of threads to use (0 = use all).
+   * \param cpus A list of cpu to use for affinity setting.
```

Comment:
@huajsj Question regarding cpu list parameter "cpus":
If I wanna split the network into two sub-graphs, then set the first graph → 4 small cores, second graph ->4 big cores. Should I set cpus =[4,4]?

Reply:
@HungYangChang , here we use a list of cpu id as parameters, if the 4 small cores have id [0, 1, 2,3] , then the first cpus list is {0, 1, 2, 3},  the command "cat /proc/cpuinfo" can be used to find cpu id information in linux.
you also can refer "TVMBackendAffinityConfigure" in "threading_backend_test.cc" for use case.